src/apps/dashboard/features/activity/components/LogLevelCell.tsx

```diff
@@ -0,0 +1,13 @@
+import type { LogLevel } from '@jellyfin/sdk/lib/generated-client/models/log-level';
+import React, { type FC } from 'react';
+
+import { ActivityLogEntryCell } from '../types/ActivityLogEntryCell';
+import LogLevelChip from './LogLevelChip';
+
+const LogLevelCell: FC<ActivityLogEntryCell> = ({ cell }) => (
+    cell.getValue<LogLevel | undefined>() ? (
+        <LogLevelChip level={cell.getValue<LogLevel>()} />
+    ) : undefined
+);
```

Comment:
`getValue` once. Worth?
```suggestion
const LogLevelCell: FC<ActivityLogEntryCell> = ({ cell }) => {
    const v = cell.getValue<LogLevel | undefined>();
    return v ? (
        <LogLevelChip level={v} />
    ) : undefined;
};
```